### ISSUE TITLE
Image-coded cache types don't work after 2017-12-15

### ISSUE THREAD

**Issue by steve8x8** [steve8x8/geotoad#354]
```
 ***  Cache image code  for  - please report
 ***  Error in parseSearchData():data.split
/.../geotoad/lib/search.rb:925:in `block in parseSearchData': undefined method `split' for nil:NilClass (NoMethodError)
```

Caused by image URLs now containing /WptTypes/ instead of /wpttypes/.

**Comment by steve8x8** [comment]
Hopefully fixed in git a30740b - release delayed once again. (Aiming at a solution that's completely case-insensitive in such contexts.)

**Comment by steve8x8** [comment]
Fixed by 3.27.2, released 2017-12-18?

**Comment by steve8x8** [comment]
.
